userspace/libsinsp/sinsp_filtercheck_k8s.cpp

```diff
@@ -36,9 +36,13 @@ static inline bool str_match_start(const std::string& val, size_t len, const cha
 
 static const filtercheck_field_info sinsp_filter_check_k8s_fields[] =
 {
+	{PT_CHARBUF, EPF_NONE, PF_NA, "k8s.ns.name", "Namespace Name", "Kubernetes namespace name."},
```

Comment:
Just moved this up so it's easier for us to maintain since it was buried in between the deprecated fields.